packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -46,6 +47,7 @@ const DEFAULT_INDEXING_DB_FILENAME = new URL('./indexing.sqlite', DEFAULT_CONFIG
  */
 const generateDefaultDaemonConfig = () => {
   const privateSeedUrl = generateSeedUrl()
+  const getIndexingConfig = getNetworkDefaultConfig(Networks.TESTNET_CLAY)
```

Comment:
can we rename this to something that makes it clearer the difference between that function and this one?  Maybe call `getNetworkDefaultConfig` "getDefaultCDBDatabaseConfig"?  And change the variable to be called "defaultDBConfig"?